internal/featureflags/temporary.go

```diff
@@ -16,62 +16,39 @@
 	// Introduced: 2024-03-29; v2.13.0
-	SkipReadOnlyAccountGroupUpdates TemporaryFlag = "MONACO_SKIP_READ_ONLY_ACCOUNT_GROUP_UPDATES"
+	SkipReadOnlyAccountGroupUpdates FeatureFlag = "MONACO_SKIP_READ_ONLY_ACCOUNT_GROUP_UPDATES"
 	// Documents toggles whether documents are downloaded and / or deployed.
 	// Introduced: 2024-04-16; v2.14.0
-	Documents TemporaryFlag = "MONACO_FEAT_DOCUMENTS"
+	Documents FeatureFlag = "MONACO_FEAT_DOCUMENTS"
 	// DeleteDocuments toggles whether documents are deleted
 	// Introduced: 2024-04-16; v2.14.2
-	DeleteDocuments TemporaryFlag = "MONACO_FEAT_DELETE_DOCUMENTS"
+	DeleteDocuments FeatureFlag = "MONACO_FEAT_DELETE_DOCUMENTS"
 	// PersistSettingsOrder toggles whether insertAfter config parameter is persisted for ordered settings.
 	// Introduced: 2024-05-15; v2.14.0
-	PersistSettingsOrder TemporaryFlag = "MONACO_FEAT_PERSIST_SETTINGS_ORDER"
+	PersistSettingsOrder FeatureFlag = "MONACO_FEAT_PERSIST_SETTINGS_ORDER"
 	// OpenPipeline toggles whether openpipeline configurations are downloaded and / or deployed.
 	// Introduced: 2024-06-10; v2.15.0
-	OpenPipeline TemporaryFlag = "MONACO_FEAT_OPENPIPELINE"
+	OpenPipeline FeatureFlag = "MONACO_FEAT_OPENPIPELINE"
 	// IgnoreSkippedConfigs toggles whether configurations that are marked to be skipped should also be excluded
 	// from the dependency graph created by Monaco. These configs are not only skipped during deployment but also
 	// not validated prior to deployment. Further, other configs cannot reference properties of this config anymore.
-	IgnoreSkippedConfigs TemporaryFlag = "MONACO_FEAT_IGNORE_SKIPPED_CONFIGS"
+	IgnoreSkippedConfigs FeatureFlag = "MONACO_FEAT_IGNORE_SKIPPED_CONFIGS"
 	// Segments toggles whether segment configurations are downloaded and / or deployed.
 	// Introduced: v2.18.0
-	Segments TemporaryFlag = "MONACO_FEAT_SEGMENTS"
+	Segments FeatureFlag = "MONACO_FEAT_SEGMENTS"
 )
 
-// Temporary FeatureFlags - for features that are hidden during development or have some uncertainty.
+// temporary FeatureFlags - for features that are hidden during development or have some uncertainty.
 // These should always be removed after release of a feature, or some stabilization period if needed.
-var Temporary = map[TemporaryFlag]FeatureFlag{
-	SkipReadOnlyAccountGroupUpdates: {
-		envName:        string(SkipReadOnlyAccountGroupUpdates),
-		defaultEnabled: false,
-	},
-	Documents: {
-		envName:        string(Documents),
-		defaultEnabled: true,
-	},
-	DeleteDocuments: {
-		envName:        string(DeleteDocuments),
-		defaultEnabled: true,
-	},
-	PersistSettingsOrder: {
-		envName:        string(PersistSettingsOrder),
-		defaultEnabled: true,
-	},
-	OpenPipeline: {
-		envName:        string(OpenPipeline),
-		defaultEnabled: true,
-	},
-	IgnoreSkippedConfigs: {
-		envName:        string(IgnoreSkippedConfigs),
-		defaultEnabled: false,
-	},
-	Segments: {
-		envName:        string(Segments),
-		defaultEnabled: false,
-	},
+var temporary = map[FeatureFlag]defaultValue{
```

Comment:
Maybe rename like this:

```suggestion
var temporaryDefaultValues = map[FeatureFlag]defaultValue{
```

Reply:
done